spec/support/helpers/feature_helpers.rb

```diff
@@ -5,8 +5,11 @@ def screenshot_after
     yield
 
     if @metadata_screenshot
+      @screenshot_index = defined?(@screenshot_index) ? @screenshot_index + 1 : 1
```

Comment:
This approach was taken from how [GCF numbers their screenshots](https://github.com/codeforamerica/gcf-backend/blob/4a41dbcd6af253bb3ea42474afa1eadd667e4ec4/spec/support/system_test_helpers.rb#L167).